File: Practice/09/Python/code.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-
a,c = map(int, input().split(':'))
a1,c1 = map(int, input().split(':'))
T1 = 60 * a + c
T2 = 60 * a1+ c1
if a > 24 or c > 60 or a1 > 24 or c1 > 60 or a < 0 or c < 0 or a1<0 or c1 < 0:
    exit(0)
if abs(T1-T2)>0 and abs(T1-T2)<16 :
    print('Встреча состоится')
else :
    print('Встреча не состоится')

File: Practice/09/C++/code.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
    int H,T,H1,T1,H2,T2,H3;
    char A;

    cin >>H>>A>>T;
    cin >>H2>>A>>T2;
    if (H>24 or H < 1){
        return 0;
    }
    if (T>60 or T <0){
        return 0;
    }

    H1 = 60*H;
    T1 = H1 + T;   
    H3 = 60*H2;
    T2 = H3 + T2;

    if (abs(T1-T2) > 15 and abs(T1-T2) < 1){
        cout << "Встреча не состоится" << endl;
        return 0;
    }

    if (abs(T1-T2) < 16 and abs(T1-T2) > 0){
        cout << "Встреча состоится" << endl;
        return 0;
    }
return 0;
}